spec/models/print_job_spec.rb

```diff
@@ -211,13 +211,86 @@
         )
 
       allow(pj).to receive(:get_label_template_by_service).and_return(label_template_name_sprint)
-      allow(SPrintClient).to receive(:send_print_request).and_return('a response')
+      response = Net::HTTPSuccess.new(1.0, '200', 'OK')
+      response.instance_variable_set(:@read, true)
+      response.instance_variable_set(
+        :@body,
+        "{\"data\":{\"print\":{\"jobId\":\"psd-2:68b27056-11cf-41ff-9b22-bdf6121a95be\"}}}"
```

Comment:
Advisory `{ data: { print: { jobId: "psd-2:68b27056-11cf-41ff-9b22-bdf6121a95be" } } }.to_json` is slightly more readable